interp/notation.ml

```diff
@@ -1357,6 +1357,7 @@ let find_with_delimiters = function
       match (String.Map.find scope !scope_map).delimiters with
         | Some key -> Some (Some scope, Some key)
         | None -> None
+        | exception Not_found -> None
```

Comment:
Why's this needed?

Reply:
As explained in the commit message, to get primitive constants printed in the debugger (rather than a Not_found exception due to the unknown int63 scope delimiter).

Reply:
Maybe this should be moved in the specific functions for printing primitive types?